.travis.yml

```diff
@@ -29,11 +26,10 @@ script:
   - NOCONFIGURE=1 ./autogen.sh
   - >
     if [ -n "$MINGW" ]; then
-      arg=-2; [ "$GTK3" = yes ] && arg=-3;
       unset CC CXX;
-      sh ./scripts/cross-build-mingw.sh $arg;
+      sh ./scripts/cross-build-mingw.sh -3;
```

Comment:
TODO: update `cross-build-mingw.sh` to not take argument and only install GTK+3 stuff.